### PR TITLE
Now move algop moves shard only if target node has less than 90% space

### PR DESCRIPTION
**Pull request by hiteshk25**

### CODE REVIEW

File: solrman/smmodel/model.go [hunk 1]
```diff
@@ -285,8 +289,8 @@ func (m *Model) computeNextMove(immobileCores []bool) *Move {
 				// no good choices
 				break
 			}
-			if target.MaxSize > 0 && target.Size >= target.MaxSize {
-				// too much data already on this node
+			if core.Size+target.Size > target.MaxSize*9/10 {
```
Comment (hiteshk25) [inline solrman/smmodel/model.go:292]: do only if target node has enough space

File: solrman/smmodel/model.go [hunk 2]
```diff
@@ -285,8 +288,7 @@ func (m *Model) computeNextMove(immobileCores []bool) *Move {
 				// no good choices
 				break
 			}
-			if target.MaxSize > 0 && target.Size >= target.MaxSize {
-				// too much data already on this node
+			if core.Size+target.Size > target.MaxSize {
```
Comment (hiteshk25) [inline solrman/smmodel/model.go:291]: We are passing 90% of disk size, thus just make after move it should cross that size

Comment (justinrsweeney) [reply solrman/smmodel/model.go:291]: Nit: This should probably retain the check for `target.MaxSize > 0` as well.

**Review by justinrsweeney — APPROVED** [review]
One minor nit, otherwise LGTM!

**Pull request merged**